docs_src/microservices/support/notifications/Ch-AlertsNotifications.md

```diff
@@ -119,48 +119,21 @@ Cleanup service removes old notification and transmission records.
 
 ## Configuration Properties
 
+Please refer to the general Configuration documentation for configuration properties common across all services.
```

Comment:
Make general Configuration documentation a link back to the general Configuration documentation